lib/sites.js

```diff
@@ -19,4 +19,21 @@ Sites.prototype.query = function( args ) {
 	});
 }
 
+Sites.prototype.getMeta = function( site_id, meta_key ) {
+	var self = this;
+
+	return new Promise( function( resolve, reject ) {
+		self.api
+			.get( '/sites/' + site_id + '/meta/' + meta_key )
+			.end( function( err, res ) {
+				if ( err ) {
+					return reject( err );
+				}
+
+				return resolve( res.body );
```

Comment:
I wonder if we should simplify this for the calling code. Currently this will return (like the .query() method) the raw response, which is not great as it means calling code must parse that every time.

If we returned the meta directly, we can go from:

```
vip.sites.getMeta( id, 'key' ).then( response => {
    const meta = response.data[ 0 ]; // wuuuut...i just want my meta bro

    // .. do stuff with meta
} );
```

to:

```
vip.sites.getMeta( id, 'key' ).then( meta => { //...do something with meta, yay } );
```

Reply:
Agreed, this needs revision. As I've started using this, returning the raw response also means logic ends up in the `catch` if the meta doesn't exist: https://github.com/Automattic/vip-go-crontab/pull/13/commits/d3a5b3cc68fbbb5096b1456650198dc96b9b52f2#L55. Not good.

Reply:
> // wuuuut...i just want my meta bro

:rofl: